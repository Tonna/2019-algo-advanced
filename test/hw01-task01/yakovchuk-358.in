25
1 1 1 2 1 1 1 1 2 1 3 5 7 5 12 5 13 16 18 4 3 2 15 13
10
6 4
23 6
4 3
23 8
16 5
10 4
11 8
2 1
17 1
12 1
