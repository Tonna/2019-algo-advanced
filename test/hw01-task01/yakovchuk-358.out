9
9
6
11
21
6
19
2
5
3
